src/components/Calculator.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import ButtonCalculator from './utilities/ButtonCalculator';
+// import InputCalculator from './utilities/InputCalculator';
+
+// styled-component
+const CalculatorStyled = styled.div`
+  width: 100%;
+  height: 500px;
+  p {
+    height: 100px;
+    font-size: 3rem;
+    font-weight: bold;
+    background-color: #A9A9A9;
+    color: #ffffff;
+    margin-top: 0;
+    margin-bottom: 0;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+  }
```

Comment:
- [ ] Please consider reducing the width of the calculator design to ensure that the design is close the wireframe. This is to ensure that the calculator has a good user interface for easy use.
- [ ] [OPTIONAL] You may consider centering the calculator to have a better and professional design ☺️.

Reply:
Sure Bro! I am Working on it.